<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeAdminInfoDialog</class>
 <widget class="QDialog" name="changeAdminInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="phone_edt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_edt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_passwd_wdt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="passwd_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返   回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>联系电话：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_name_edt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="passwd_again_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="id_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>编号：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwd_edt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="username_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保   存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
